content/adapters/pusher.md

```diff
@@ -58,7 +58,7 @@ Please note:
 
 - While the Pusher adapter is quite a light translation layer (certainly a good deal lighter than the Pubnub adapter), a protocol adapter inevitably adds some latency. Using the adapter will be a little slower than using Ably native client libraries. Typically the impact is in the low milliseconds. By the same token, it will likely also be marginally slower than using Pusher natively -- but only if you are close to whichever Pusher datacenter you are using. If not, the extra milliseconds of latency from the adapter should be more than compensated for by being able to use a datacenter closer to you (unlike Pusher, Ably has [16+ datacenters globally and 175+ edge acceleration points-of-presence](https://ably.com/network) and federates messages between them - you automatically connect to the closest one to you, using latency-based routing).
 - Behind the scenes, the adapter just uses the normal Ably service, so there is no problem with using Pusher and Ably client libraries side by side (though bear in mind channel name translation, see below). You can mix and match as you like; for example, using Pusher client libraries normally, but using the Ably REST api to get channel history, which is not available through the Pusher clients.
-- While using the adapter gives you some of the advantages of Ably over Pusher (eg inter-region message federation), many others (e.g. [continuity guarantees](https://faqs.ably.com/connection-state-recovery), [fallback host support](https://faqs.ably.com/routing-around-network-and-dns-issues), [history](https://ably.com/docs/realtime/history), [flexible channel namespaces](https://faqs.ably.com/what-is-a-channel-namespace-and-how-can-i-use-them), [powerful token authentication](https://ably.com/docs/core-features/authentication)) require the use of the Ably client libraries. As a result, if a native Ably library is available for your platform, we recommend you consider using the Ably client libraries instead or at least make a plan to eventually transition over to Ably native client libraries.
+- While using the adapter gives you some of the advantages of Ably over Pusher (eg inter-region message federation), many others (e.g. [continuity guarantees](https://faqs.ably.com/connection-state-recovery), [fallback host support](https://faqs.ably.com/routing-around-network-and-dns-issues), [history](https://ably.com/docs/storage-history/history), [flexible channel namespaces](https://faqs.ably.com/what-is-a-channel-namespace-and-how-can-i-use-them), [powerful token authentication](https://ably.com/docs/core-features/authentication)) require the use of the Ably client libraries. As a result, if a native Ably library is available for your platform, we recommend you consider using the Ably client libraries instead or at least make a plan to eventually transition over to Ably native client libraries.
```

Comment:
@StevenAppleby  it looks like the majority of these changes are to the file path for storage history but that Mark's comment's on the history file haven't been addressed. Please review those and then we can push this branch with both the content changes and the path changes through.